apps/condo/cypress/plugins/index.js

```diff
@@ -54,6 +57,50 @@ module.exports = async (on, config) => {
 
             return userObject
 
+        },
+        async 'keystone:createSupportWithProperty' () {
+            if (isEmpty(supportObject)) {
+                const client = await makeClientWithRegisteredOrganization()
+                const property = await createTestProperty(
+                    client,
+                    client.organization,
+                    { map: buildingMapJson },
+                    false, {
+                        city: 'Екатеринбург',
```

Comment:
Why not faker values?

Reply:
This address writed in tables for ticket import. If will be fake address name then import will be not work - in organization will have not needed address as in tables

Reply:
can you take this value from testFiles?

Reply:
I don't think it's worth it now. Cypress tests for ticket import work correctly only locally. This may take a lot of time